2021-05-18 12:17:19,017 INFO    MainThread:173586 [wandb_setup.py:_flush():69] setting env: {}
2021-05-18 12:17:19,017 INFO    MainThread:173586 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-18 12:17:19,017 INFO    MainThread:173586 [wandb_init.py:_log_setup():336] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/American-Sign-Language-Recognition/wandb/run-20210518_121719-24a5ffei/logs/debug.log
2021-05-18 12:17:19,017 INFO    MainThread:173586 [wandb_init.py:_log_setup():337] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/American-Sign-Language-Recognition/wandb/run-20210518_121719-24a5ffei/logs/debug-internal.log
2021-05-18 12:17:19,017 INFO    MainThread:173586 [wandb_init.py:init():369] calling init triggers
2021-05-18 12:17:19,017 INFO    MainThread:173586 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-18 12:17:19,018 INFO    MainThread:173586 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 1wmxhbg3
2021-05-18 12:17:19,019 INFO    MainThread:173586 [wandb_run.py:finish():1208] finishing run ranuga-d/Sign-Language-Recognition/1wmxhbg3
2021-05-18 12:17:19,024 INFO    MainThread:173586 [jupyter.py:save_history():349] saving 47 cells to _session_history.ipynb
2021-05-18 12:17:19,034 INFO    MainThread:173586 [wandb_run.py:_config_callback():787] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2021-05-18 12:17:19,058 INFO    MainThread:173586 [jupyter.py:save_ipynb():268] looking for notebook: main.ipynb
2021-05-18 12:17:20,099 INFO    MainThread:173586 [wandb_init.py:_jupyter_teardown():267] saved code and history
2021-05-18 12:17:20,099 INFO    MainThread:173586 [wandb_init.py:_jupyter_teardown():268] cleaning up jupyter logic
2021-05-18 12:17:20,099 INFO    MainThread:173586 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-18 12:17:20,099 INFO    MainThread:173586 [wandb_run.py:_restore():1507] restore
2021-05-18 12:17:25,102 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: None
2021-05-18 12:17:26,279 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 131802
  total_bytes: 131802
}

2021-05-18 12:17:27,023 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 131802
  total_bytes: 131802
}

2021-05-18 12:17:28,000 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 131802
  total_bytes: 178186
}

2021-05-18 12:17:28,800 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 131802
  total_bytes: 178186
}

2021-05-18 12:17:28,901 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 131802
  total_bytes: 178186
}

2021-05-18 12:17:29,003 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 177802
  total_bytes: 178186
}

2021-05-18 12:17:29,105 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 178186
  total_bytes: 178186
}

2021-05-18 12:17:29,208 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 178186
  total_bytes: 178186
}

2021-05-18 12:17:29,309 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 178186
  total_bytes: 178186
}

2021-05-18 12:17:29,411 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 178186
  total_bytes: 178186
}

2021-05-18 12:17:29,512 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 178186
  total_bytes: 178186
}

2021-05-18 12:17:29,614 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 178186
  total_bytes: 178186
}

2021-05-18 12:17:29,716 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 178186
  total_bytes: 178186
}

2021-05-18 12:17:29,817 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 178186
  total_bytes: 178186
}

2021-05-18 12:17:31,119 INFO    MainThread:173586 [wandb_run.py:_show_files():1879] logging synced files
2021-05-18 12:17:31,122 INFO    MainThread:173586 [wandb_init.py:init():418] starting backend
2021-05-18 12:17:31,122 INFO    MainThread:173586 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-18 12:17:31,124 INFO    MainThread:173586 [backend.py:ensure_launched():132] starting backend process...
2021-05-18 12:17:31,160 INFO    MainThread:173586 [backend.py:ensure_launched():137] started backend process with pid: 235178
2021-05-18 12:17:31,161 INFO    MainThread:173586 [wandb_init.py:init():423] backend started and connected
2021-05-18 12:17:31,174 INFO    MainThread:173586 [wandb_init.py:init():465] updated telemetry
2021-05-18 12:17:31,176 INFO    MainThread:173586 [wandb_init.py:init():484] communicating current version
2021-05-18 12:17:32,150 INFO    MainThread:173586 [wandb_init.py:init():489] got version response 
2021-05-18 12:17:32,151 INFO    MainThread:173586 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-18 12:17:32,914 INFO    MainThread:173586 [wandb_init.py:init():522] starting run threads in backend
2021-05-18 12:17:37,924 INFO    MainThread:173586 [wandb_run.py:_console_start():1565] atexit reg
2021-05-18 12:17:37,924 INFO    MainThread:173586 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-05-18 12:17:37,924 INFO    MainThread:173586 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-05-18 12:17:37,924 INFO    MainThread:173586 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-18 12:17:37,924 INFO    MainThread:173586 [wandb_init.py:init():546] run started, returning control to user process
2021-05-18 12:22:50,554 INFO    MainThread:173586 [wandb_run.py:finish():1208] finishing run ranuga-d/Sign-Language-Recognition/24a5ffei
